jujugui/static/gui/src/app/app.js

```diff
@@ -995,7 +995,10 @@ YUI.add('juju-gui', function(Y) {
       // XXX j.c.sackett 2017-01-30 Right now USSO link is using
       // loginToController, while loginToAPIs is used by the login form.
       // We want to use loginToAPIs everywhere since it handles more.
-      const controllerAPI = this.controllerAPI;
+      let controllerAPI = this.env;
```

Comment:
I am worried that this approach would be confusing on the long run. In Juju1 there is no controller. The current approach of having a controllerAPI and an env, the former being null in Juju1 seems more parsable to me, as it reflects the Juju1 architecture more. So I'd keep it and I'd just fix the parts of the code wrongly assuming a controller where it could be null.